main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 	var config *rest.Config
 
 	if k8s_port := os.Getenv("KUBERNETES_PORT"); k8s_port == "" {
-		fmt.Println("Using local kubeconfig")
+		fmt.Fprintf(os.Stderr,("Using local kubeconfig\n"))
```

Comment:
Hi @korkin25, I like the PR but both of these messages are informational so would actually be `stdout`, right?

Reply:
yes, but information message makes bad json your software makes. just run go run main.go | jq '.'
the same will happen with fluentbit/etc

Reply:
I'm not sure I understand what you mean but it sounds like the correct solution would be to send these messages to `stdout` like I said but print them as JSON?

Reply:
Yes, JSON will be better. Thank you.

Reply:
Cool. Let me know when the PR is ready 🙂